doc/antora/modules/reference/nav.adoc

```diff
@@ -120,123 +120,121 @@
-**** xref:sites-available/copy-acct-to-home-server.adoc[Copy ACCT to Home Server]
-**** xref:sites-available/decoupled-accounting.adoc[Decoupled Accounting]
-**** xref:sites-available/detail.adoc[Detail]
-**** xref:sites-available/dhcp.adoc[Dhcp]
-**** xref:sites-available/dhcp.relay.adoc[Dhcp Relay]
-**** xref:sites-available/dynamic-clients.adoc[Dynamic Clients]
-**** xref:sites-available/example.adoc[Example]
-**** xref:sites-available/inner-tunnel.adoc[Inner Tunnel]
-**** xref:sites-available/ldap_sync.adoc[LDAP Sync]
-**** xref:sites-available/originate-coa.adoc[Originate CoA-Request packets]
-**** xref:sites-available/proxy-inner-tunnel.adoc[Proxy Inner Tunnel]
-**** xref:sites-available/radius-acct.adoc[Radius Acct]
-**** xref:sites-available/robust-proxy-accounting.adoc[Robust Proxy Accounting]
-**** xref:sites-available/status.adoc[Status]
-**** xref:sites-available/tacacs.adoc[Tacacs]
-**** xref:sites-available/default.adoc[The default Virtual Server]
-**** xref:sites-available/tls-cache.adoc[TLS Cache]
-**** xref:sites-available/tls.adoc[TLS]
-**** xref:sites-available/virtual.example.com.adoc[virtual.example.com]
-**** xref:sites-available/vmps.adoc[VMPS]
-**** xref:experimental.conf.adoc[Experimental modules]
-
-*** xref:clients.conf.adoc[Client Definitions]
-*** xref:debug.conf.adoc[Debugging configuration]
-*** xref:dictionary.adoc[Local dictionary definitions]
-*** xref:radrelay.conf.adoc[Radrelay Configuration]
-*** xref:radiusd.conf.adoc[Server Configuration File]
-*** xref:templates.conf.adoc[Templates]
-*** xref:trigger.conf.adoc[Triggers]
+//*** xref:raddb/global.d/python.adoc[python]
+
+** xref:raddb/mods-available/index.adoc[Modules]
+*** xref:raddb/mods-available/all_modules.adoc[Summary of all modules]
+*** xref:raddb/mods-available/abfab_psk_sql.adoc[ADFAB PSK]
+*** xref:mods-available/always.adoc[Always]
+*** xref:mods-available/attr_filter.adoc[Attr_filter]
+*** xref:mods-available/cache.adoc[Cache]
+*** xref:mods-available/cache_eap.adoc[Cache EAP]
+*** xref:mods-available/cache_tls.adoc[Cache TLS Session]
+*** xref:mods-available/chap.adoc[CHAP module]
+*** xref:mods-available/cipher.adoc[Cipher]
+*** xref:mods-available/client.adoc[Client]
+*** xref:mods-available/couchbase.adoc[Couchbase]
+*** xref:mods-available/csv.adoc[CSV]
+*** xref:mods-available/cui.adoc[CUI]
+*** xref:mods-available/date.adoc[Date]
+*** xref:mods-available/delay.adoc[Delay]
+*** xref:mods-available/detail.adoc[Detail]
+*** xref:mods-available/detail.example.com.adoc[Detail (Sample)]
+*** xref:mods-available/detail.log.adoc[Detail (Log Sample)]
+*** xref:mods-available/dhcpv4.adoc[DHCPv4]
+*** xref:mods-available/dict.adoc[Dict]
+*** xref:mods-available/digest.adoc[Digest]
+*** xref:mods-available/eap.adoc[EAP]
+*** xref:mods-available/eap_inner.adoc[EAP/Inner]
+*** xref:mods-available/echo.adoc[Echo]
+*** xref:mods-available/escape.adoc[Escape]
+*** xref:mods-available/etc_group.adoc[etc_group]
+*** xref:mods-available/exec.adoc[Exec]
```

Comment:
If one of the links needs to change, then all of the other ones should be double-checked.

i.e. it's easier to do a global search and replace of `xref:` to `xref:raddb/`.  This doesn't have to be done for everything in the file.  But a text editor should allow you to approve or reject individual changes.  in which case you can just run down the file going "yes" or "no" as needed.